PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/forms/dsl/FormBuilder.kt

```diff
@@ -87,10 +88,13 @@ class FormBuilder(var columnTemplate: ColumnTemplate? = null) {
 
     fun createGridLayoutManager() = GridLayoutManager(rowSize, colSize)
 
-    fun buildPanel(): JPanel = JPanel().apply {
+    fun buildPanel(serviceName: String): JPanel = JPanel().apply {
```

Comment:
May not be a good solution to add into a tool class. Let's consider do it in component extend functions.